src/hooks/useUniversalRouter.ts

```diff
@@ -80,6 +80,10 @@ export function useUniversalRouterSwapCallback(
         } catch (gasError) {
           setTraceStatus('failed_precondition')
           setTraceError(gasError)
+          sendAnalyticsEvent(SwapEventName.SWAP_ESTIMATE_GAS_CALL_FAILED, {
+            ...formatCommonPropertiesForTrade(trade, options.slippageTolerance),
```

Comment:
We'll need the tx request object here to see why it failed - can you include that too?

Reply:
also the `gasError` object if possible